configure.ac

```diff
@@ -180,6 +180,16 @@ else
     fi
 fi
 
+# Get the size of long int
+AC_CACHE_CHECK([Size of long int],
+[iperf3_cv_sizeof_long],
+AC_RUN_IFELSE(
+  [AC_LANG_PROGRAM([[#include <stdio.h>]],
+                   [[fprintf(stderr, "%d", sizeof(long)); return(0)]])],
```

Comment:
Thanks for the pull request! I got a compiler warning with macOS / Clang on this line (line 188) where it complained of a mismatch between the `%d` format specifier (which takes an int) and `sizeof(long)`, which is a long. If I changed the `%d` to `%zd` then this seems to work correctly.

Reply:
That's all kinda pointless.
`<inttypes.h>` is in c99, c99 is supported everywhere (that's why issue gone unnoticed for 10 years), and `"%zd"` is from c99 too. Besides, `AC_RUN_IFELSE` breaks on cross-compilation (you need check that fails on compilation stage), etc.
Just use `<inttypes.h>' unconditionally and be done.